jembi/openhim-core-js
# Npm package

Closes #282 

@devcritter can you please review this. Also, @BMartinos @debonair @armageddon this may be of interest.

To test this run:

`sudo npm install -g .` form the root directory of the source.

Then run `openhim-core` (from anywhere, it should be put in your path) and the server should startup, with forever :)

Also, to symlink you current working directory to the package run `npm link`, then any changes you make are instantly accessible via the binary. Also, forever is set to watch files so as soon as it sees newly compiled files it will restart the server!
